docs/PacketFence_Checkpoint_Quick_Install_Guide-docinfo.xml

```diff
@@ -0,0 +1,24 @@
+<copyright>
+  <year>2014</year>
+  <holder>Inverse inc.</holder>
+</copyright>
+
+<revhistory>
+  <revision>
+    <revnumber>1.0</revnumber>
+    <date>2014-10-20</date>
```

Comment:
Maybe we can fit the date with the pull-request merge date (give or take... at least make it the same year as the copyright :))